### PR TITLE
Fix privilege

### PR DESCRIPTION
**Pull request by pokharelmanish**
### Jira Story

- [Fix the sensitive and sealed issue](https://osi-cwds.atlassian.net/browse/SNAP-984)

## Description
Fix the sensitive and sealed issue. 

## Tests
- [x] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have ran all tests for the project.
- [ ] I have ran lint/rubocop check for the changed/new files.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/common/search/SearchResultsTable.jsx [hunk 1]
```diff
@@ -39,7 +38,7 @@ class SearchResultsTable extends Component {
         const className = person.isSealed ? 'disabled-cursor' : ''
         return (
           <div>
-            {person.isSensitive ? <Link className='sensitive-person' onClick={sensitiveAlert}>{person.fullName}</Link> : <Link className={className} to={`/snapshot/detail/${id}`}>{person.fullName}</Link>}
+            {<Link className={className} to={`/snapshot/detail/${id}`}>{person.fullName}</Link>}
```
Comment (GajulaNagaPradeep) [inline app/javascript/common/search/SearchResultsTable.jsx]: This constant is really vague. Should be more specific.

Comment (pokharelmanish) [reply app/javascript/common/search/SearchResultsTable.jsx]: which constant?

File: app/javascript/common/search/SearchResultsTable.jsx [hunk 2]
```diff
@@ -8,7 +8,6 @@ import {Link} from 'react-router'
 import ReactTooltip from 'react-tooltip'
 import {phoneNumberFormatter} from 'utils/phoneNumberFormatter'
 
-const sensitiveAlert = () => alert('You are not authorized to add this person.') // eslint-disable-line no-alert
```
Comment (ggomez-cambriasolutions) [inline app/javascript/common/search/SearchResultsTable.jsx]: could this method be moved inside the class?

Comment (pokharelmanish) [reply app/javascript/common/search/SearchResultsTable.jsx]: Actually, I removed it.

Comment (ggomez-cambriasolutions) [reply app/javascript/common/search/SearchResultsTable.jsx]: oh ok, you're right.

**Pull request merged**